test/packages/EABase/include/Common/EABase/config/eacompiler.h

```diff
@@ -726,9 +726,9 @@
 			// Extern template is supported.
 		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__EDG_VERSION__) && (__EDG_VERSION__ >= 401)  // EDG 4.1+.
 			// supported.
-		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__APPLE__) && (EA_COMPILER_VERSION >= 401)
+		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__apple_build_version__) && (EA_COMPILER_VERSION >= 401)
```

Comment:
Ah you know what I just did a s/__APPLE__/__apple_build_version__/ on this file because I saw a couple bad examples. Let me revisit... In hindsight, I think the combination of __APPLE__ __clang__ is clearer than simply __apple_build_version__.